js/pro/woo.js

```diff
@@ -0,0 +1,722 @@
+        //         data: {
+        //             symbol: 'PERP_BTC_USDT',
+        //             bids: [
+        //                 [
+        //                     0.30891,
+        //                     2469.98
+        //                 ]
+        //             ],
+        //             asks: [
+        //                 [
+        //                     0.31075,
+        //                     2379.63
+        //                 ]
+        //             ]
+        //         }
+        //     }
+        //
+        const data = this.safeValue (message, 'data');
+        const marketId = this.safeString (data, 'symbol');
+        const market = this.safeMarket (marketId);
+        const symbol = market['symbol'];
+        const topic = this.safeString (message, 'topic');
+        let orderbook = this.safeValue (this.orderbooks, symbol);
+        if (orderbook === undefined) {
+            orderbook = this.orderBook ({});
+        }
+        const timestamp = this.safeInteger (message, 'ts');
+        const snapshot = this.parseOrderBook (data, symbol, timestamp, 'bids', 'asks');
+        orderbook.reset (snapshot);
+        client.resolve (orderbook, topic);
+    }
+
+    async watchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
+        const name = 'ticker';
+        const market = this.market (symbol);
+        const topic = market['id'] + '@' + name;
+        const request = {
+            'event': 'subscribe',
+            'topic': topic,
+        };
+        const message = this.extend (request, params);
+        return await this.watchPublic (topic, message);
+    }
+
+    parseTicker (ticker, market = undefined) {
+        //
+        //     {
+        //         symbol: 'PERP_BTC_USDT',
+        //         open: 19441.5,
+        //         close: 20147.07,
+        //         high: 20761.87,
+        //         low: 19320.54,
+        //         volume: 2481.103,
+        //         amount: 50037935.0286,
+        //         count: 3689
+        //     }
+        //
+        const timestamp = this.safeInteger (ticker, 'date', this.milliseconds ());
+        return this.safeTicker ({
+            'symbol': this.safeSymbol (undefined, market),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeString (ticker, 'high'),
+            'low': this.safeString (ticker, 'low'),
+            'bid': undefined,
+            'bidVolume': undefined,
+            'ask': undefined,
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': this.safeString (ticker, 'open'),
+            'close': this.safeString (ticker, 'close'),
+            'last': undefined,
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': undefined,
+            'average': undefined,
+            'baseVolume': this.safeString (ticker, 'volume'),
+            'quoteVolume': undefined,
```

Comment:
isn't the amount the quote volume? 
![image](https://user-images.githubusercontent.com/43336371/205095042-b34186fd-7559-437f-b13c-e56866e9b793.png)